file-mod/cmd/replace.go

```diff
@@ -0,0 +1,27 @@
+package cmd
+
+import (
+	"github.com/chef/ci-studio-common/lib"
+
+	"github.com/spf13/cobra"
+)
+
+var replaceCmd = &cobra.Command{
+	Use:   "find-and-replace REGEX_STR STRING FILE",
+	Short: "Replace REGEX_STR with STRING in FILE. Supports multiline replace.",
+	Args:  cobra.MinimumNArgs(3),
+	Run:   findAndReplace,
```

Comment:
ℹ️ There's also a `RunE` that will exit with a non-zero exit code if the passed function returns an error != nil.